src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
In the `@font-face` directive, the `font-family` property should only contain the name of the font, not the fallback fonts. Fallback fonts should be specified in the CSS rules where the font is being used. So, it should be `font-family: Avenir;`.